packages/@glimmer/integration-tests/test/initial-render-test.ts

```diff
@@ -277,11 +309,8 @@ class Rehydration extends AbstractRehydrationTests {
     this.element = assertElement(host.firstChild);
 
     this.renderClientSide(template, { remote: clientRemote });
-    this.assertRehydrationStats({ nodesRemoved: 1 });
-    this.assert.equal(
-      toInnerHTML(clientRemote),
-      '<prefix></prefix><suffix></suffix><inner>Wat Wat</inner>'
-    );
+    this.assertRehydrationStats({ nodesRemoved: 2 });
```

Comment:
Using `{{in-element` **without** passing `insertBefore` as a named argument _always_ means "clear this element".

* https://github.com/emberjs/rfcs/blob/master/text/0287-promote-in-element-to-public-api.md#small-proposed-changes
* https://github.com/glimmerjs/glimmer-vm/pull/918/files